[2025-03-19T18:41:50.613+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-19T18:41:50.668+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_Api_v1.get_name scheduled__2023-12-04T00:00:00+00:00 [queued]>
[2025-03-19T18:41:50.679+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_Api_v1.get_name scheduled__2023-12-04T00:00:00+00:00 [queued]>
[2025-03-19T18:41:50.679+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-03-19T18:41:50.727+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): get_name> on 2023-12-04 00:00:00+00:00
[2025-03-19T18:41:50.781+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1110) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-19T18:41:50.785+0000] {standard_task_runner.py:72} INFO - Started process 1122 to run task
[2025-03-19T18:41:50.783+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_Api_v1', 'get_name', 'scheduled__2023-12-04T00:00:00+00:00', '--job-id', '213', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_taskflow_api.py', '--cfg-path', '/tmp/tmpbuexjnaj']
[2025-03-19T18:41:50.790+0000] {standard_task_runner.py:105} INFO - Job 213: Subtask get_name
[2025-03-19T18:41:50.972+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_taskflow_Api_v1.get_name scheduled__2023-12-04T00:00:00+00:00 [running]> on host 63fd4b143e0a
[2025-03-19T18:41:51.274+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_Api_v1' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2023-12-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-04T00:00:00+00:00'
[2025-03-19T18:41:51.281+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-19T18:41:51.287+0000] {python.py:240} INFO - Done. Returned value was: Jerry
[2025-03-19T18:41:51.383+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-19T18:41:51.392+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_Api_v1, task_id=get_name, run_id=scheduled__2023-12-04T00:00:00+00:00, execution_date=20231204T000000, start_date=20250319T184150, end_date=20250319T184151
[2025-03-19T18:41:51.608+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-19T18:41:51.782+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-19T18:41:51.816+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
